### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by mpagot**
Cite the config.yaml and the glue script in the main README Fix typos in the playbooks documenation
Add playbook document in regression

### CODE REVIEW

**Comment by lpalovsky** [comment]
Looks good so far.

**Review by alvarocarvajald — APPROVED** [review]
LGTM

**Review by lpalovsky — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -32,7 +32,38 @@ ssh-add id_rsa_cloud
 
 ### Build and Destroy Terraform and Ansible components
 
-The build and destroy scripts are still an early stage of development and represent the progress made so far.
+To get started you must should create a yml configuration file
```
Comment (alvarocarvajald) [inline README.md:35]: It should be either
> you should create

or
> you must create

File: README.md [hunk 2]
```diff
@@ -32,7 +32,38 @@ ssh-add id_rsa_cloud
...
+
+```shell
+cp config.yaml.example config.yaml
+```
+
+Edit the values of `config.yaml` to match your configuration.
+
+Run the `config` step to get all the needed Terraform and Ansible configuration file generated.
```
Comment (alvarocarvajald) [inline README.md:43]: file**s**

**Review by alvarocarvajald — APPROVED** [review]
Just two minor typos. Otherwise LGTM

**Pull request merged**